Traceback (most recent call last):
  File "nltk/nltk-pos.py", line 5, in <module>
    from python.nltk.get_folder import getFolder
ImportError: No module named 'python'
Command exited with non-zero status 1
	Command being timed: "python3.5 nltk/nltk-pos.py"
	User time (seconds): 0.50
	System time (seconds): 1.10
	Percent of CPU this job got: 345%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:00.46
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 67212
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 11870
	Voluntary context switches: 63
	Involuntary context switches: 660645
	Swaps: 0
	File system inputs: 0
	File system outputs: 8
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
